sdk/notificationhubs/notification-hubs/CHANGELOG.md

```diff
@@ -4,10 +4,27 @@
 
 ### Features Added
 
+- Added implementation for `enableTestSend` to gather detailed statistics.
+
 ### Breaking Changes
 
+The largest breaking change comes to GCM/FCM compatibility.  Azure Notification Hubs does not support the FCM V1 API, and instead is using the Firebase Legacy HTTP API.  As the GCM name is considered obsolete, we are renaming GCM specific things to `FcmLegacy` across the board, except for the XML types such as `GcmRegistrationDescription` and `GcmTemplateRegistrationDescription`.  Helper methods are created for the registration descriptions to be `createFcmLegacyRegistrationDescription` and `createFcmLegacyTemplateRegistrationDescription`.  Azure Notification Hubs will eventually support the V1 API and we are reserving the `FcmRegistrationDescription` and related classes for this functionality.
+
+List of breaking changes:
+
+- FCM/GCM Compatibility:
+  - Removed `FcmRegistrationDescription` and `FcmTemplateRegistrationDescription`
+  - Removed deprecations from `GcmRegistrationDescription` and `GcmTemplateRegistrationDescription`
+  - Created `createFcmLegacyRegistrationDescription` and `createFcmLegacyTemplateRegistrationDescription`
+  - `FirebaseLegacyNotification` and associated factory method renamed to `FcmLegacyNotification`
+  - `FirebaseLegacyInstallation` and associated factory method renamed to `FcmLegacyInstallation`
+- `enableTestSend` option is no longer available for `sendDirectNotification`.
+
 ### Bugs Fixed
 
+- #23065
+- #23089
```

Comment:
Can you please add brief descriptions of the issues fixed?

Reply:
Done